### PR TITLE
feat: mark some s390x compute tests as working

### PR DESCRIPTION
**Pull request by nestoracunablanco**

### CODE REVIEW

File: tests/virt/node/efi_secureboot/test_vm_with_efi_secureboot.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 @pytest.mark.gating
 @pytest.mark.polarion("CNV-4465")
+@pytest.mark.s390x
```
[the remark this answers is not in the document]
Comment (chandramerla) [reply tests/virt/node/efi_secureboot/test_vm_with_efi_secureboot.py:17]: If we agree on this proposal to be consistent - we've to mark the test whichever we've marked as `s390x` as `x86_64` as well. All the existing tests I've assumed would work for s390x, unless if there are any corner cases.

File: tests/virt/node/general/test_static_ssh_key_injection.py [hunk 2]
```diff
@@ -77,6 +77,7 @@ class TestVMWithStaticKeyInjection:
     def test_static_ssh_key_injection(self, vm_with_ssh_secret):
         running_vm(vm=vm_with_ssh_secret)
 
+    @pytest.mark.s390x
```
Comment (chandramerla) [inline tests/virt/node/general/test_static_ssh_key_injection.py:80]: Actually when I've run node gating, on s390x branch, both the tests in this class are working.
So, better we add the s390x marker at the class level itself.

Is `test_static_ssh_key_injection` failing for you?

File: tests/virt/cluster/common_templates/centos/test_centos_os_support.py [hunk 3]
```diff
@@ -183,6 +183,7 @@ def test_verify_virtctl_guest_agent_data_after_migrate(
             vm=golden_image_vm_object_from_template_multi_centos_multi_storage_scope_class
         ), "Guest agent stopped responding"
 
+    @pytest.mark.s390x
```
[the remark this answers is not in the document]
Comment (nestoracunablanco) [reply tests/virt/cluster/common_templates/centos/test_centos_os_support.py]: Done

**Comment by nestoracunablanco** [comment]
/verified
https://jenkins-csb-cnvqe-main.dno.corp.redhat.com/job/test-pytest-cnv-4.19-virt-cluster-s390x/5/
